kolena/_utils/validators.py

```diff
@@ -20,3 +22,9 @@ class ValidatorConfig:
     arbitrary_types_allowed = True
     smart_union = True
     extra = Extra.allow  # do not fail when unrecognized values are provided
+
+
+def validate_name(field: str, field_name: Optional[str] = None):
+    field_name_str = field_name if field_name else "field"
+    if not bool(field and not field.isspace()):
```

Comment:
Why is this so complicated? Why not just:
```
if field.isspace():
    ...
```

If you're also hoping to validate that `field` is non-None, the `@validate_arguments` would probably work.